ldap_reader/reader.py

```diff
@@ -176,6 +183,12 @@ def _build_user_dict(self, result_dict):
             self.config,
             enabled_attrs_dict)
 
+        # Add 'uniqueid'
+        guid_source = self.config.get('dir_guid_source', None)
```

Comment:
We should enforce that `dir_guid_source` exists. Current NetKES usage depends on its existence, as does plans for Semaphor usage. If it doesn't exist, that's an error.